api/v1/qdrantentity_types.go

```diff
@@ -74,8 +74,8 @@ type QdrantEntityStatus struct {
 	Phase EntityPhase `json:"phase,omitempty"`
 	// Result is the last result from the invocation to a manager
 	Result QdrantEntityStatusResult `json:"result,omitempty"`
-	// Timestamp when the status was last updated.
-	LastUpdatedAt metav1.MicroTime `json:"lastUpdatedAt,omitempty"`
+	// ObservedGeneration is the entity.Generation the Status is related to
```

Comment:
can you rephrase the doc? - it is a bit unclear.

Typically it is something like: `ObservedGeneration is the 'Generation' of the resource that was last processed by the controller.`